.github/actions/setup-conda/action.yml

```diff
@@ -24,14 +24,13 @@ runs:
       if: ${{ inputs.pyarrow-version }}
 
     - name: Install ${{ inputs.environment-file }}
-      uses: conda-incubator/setup-miniconda@v2.1.1
+      uses: mamba-org/provision-with-micromamba@v12
       with:
         environment-file: ${{ inputs.environment-file }}
-        activate-environment: ${{ inputs.environment-name }}
-        python-version: ${{ inputs.python-version }}
-        channel-priority: ${{ runner.os == 'macOS' && 'flexible' || 'strict' }}
+        environment-name: ${{ inputs.environment-name }}
+        extra-specs: |
+          python =${{ inputs.python-version }}
```

Comment:
So this is to specifically support the sdist workflow? (might be good to add a comment to that affect if so)

Reply:
Yes, actually come to think of it I think it's better to simply pass through `extra-specs`. It's more flexible.